xml/containerized-kubernetes.xml

```diff
@@ -87,14 +103,14 @@
   <para>
    The &rook; orchestrator uses configuration files in YAML format called
    <emphasis>manifests</emphasis>. The manifests you need are included in the
-   <package>rook-k8s-yaml</package> RPM package. Install it by running
+   <package>rook-k8s-yaml</package> RPM package. You can find this package
```

Comment:
nope. rook-k8s-yaml package is in SES6 repo. /usr/share/k8s-yaml is the path where you would see Rook manifest after installation of that package.

Reply:
@denisok what's the path? (is there a path?)

Reply:
so, section 5.3 of deployment guide says:
...
 > The following procedure describes the cluster preparation in detail. 
...
> Install and register SUSE Linux Enterprise Server 15 SP1 together with the SUSE Enterprise Storage 6 extension on each node of the cluster. 

In case of CaaSP, we don't need SLE15 SP! as we are saying:
>  A running SUSE CaaS Platform 4 cluster. 
which already implies SLE15SP1 and CaaSP repos.

But what we still need is "SUSE Enterprise Storage 6 extension"
That should be configured on kubernetes management node.
...

So we need to say that on management node these 2 repos needs to be configured too install rook-k8s-yaml package:
SUSE-Enterprise-Storage-6-Pool
 SUSE-Enterprise-Storage-6-Updates

How to configure those repos, I haven't find that in a Deployment or other guide. That seems to be more high level SUSE document. It could be done via SCC or by adding repositories.
6.8.1 has some definition but installing from DVD, which is not applicable here.